src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsFormatDefinition.cs

```diff
@@ -17,23 +17,23 @@ namespace Microsoft.CodeAnalysis.Editor.InlineHints
     internal sealed class ClassificationTypeDefinitions
     {
         [Export]
-        [Name(InlineParameterNameHintsTag.TagId)]
+        [Name(InlineHintsTag.TagId)]
         [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
-        internal ClassificationTypeDefinition InlineParameterNameHints;
+        internal ClassificationTypeDefinition InlineHints;
 
         [Export(typeof(EditorFormatDefinition))]
-        [Name(InlineParameterNameHintsTag.TagId)]
+        [Name(InlineHintsTag.TagId)]
         [Order(After = LanguagePriority.NaturalLanguage, Before = LanguagePriority.FormalLanguage)]
         [UserVisible(true)]
-        internal sealed class InlineParameterNameHintsFormatDefinition : EditorFormatDefinition
+        internal sealed class InlineHintsFormatDefinition : EditorFormatDefinition
         {
             [ImportingConstructor]
             [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-            public InlineParameterNameHintsFormatDefinition()
+            public InlineHintsFormatDefinition()
             {
-                this.DisplayName = EditorFeaturesResources.Inline_Parameter_Name_Hints;
-                this.ForegroundBrush = Brushes.Black;
-                this.BackgroundBrush = Brushes.LightGray;
+                this.DisplayName = EditorFeaturesResources.Inline_Hints;
+                this.ForegroundBrush = new SolidColorBrush(Color.FromRgb(104, 104, 104));
+                this.BackgroundBrush = new SolidColorBrush(Color.FromRgb(230, 230, 230));
```

Comment:
these match the values we have in the xml files.  This means that we get these as the defaults no matter what.